content/git/index.md

```diff
@@ -159,6 +197,20 @@ Bad Example:
   $ git merge my-branch
   ```
 
+## Pull Requests
+
+On most projects, you will typically use the source control platform (e.g. Github, Bitbucket, Azure DevOps)
+to merge branches through pull requests, rather than using the `git merge` command in a terminal.
+Pull requests should contain all the information necessary for another person to quickly understand,
+test, and review the code changes. We _strongly_ encourage the use of pull request templates, so that
+your entire team can create consistent pull requests that include all of the necessary information.
+A good pull request will typically include the following:
+
+* A summary of what the changes are and why they are being made
+* Testing instructions
+* Link(s) to any related stories or tickets
+* Screenshots or screen recordings for UI/UX changes
+* A checklist for the creator to ensure any required steps are taken (e.g. running tests)
 ## Misc.
```

Comment:
I very much prefer setting up CI/CD so that pull requests don't have to contain step-by-step instructions for cloning the code, building the site, starting the site, and then finally viewing it locally.  I wonder if a statement here to that end would be nice: "If you are having to write a paragraph of instructions to the reviewer on how they can test the code, consider augmenting your CI/CD pipeline to speed the process of reviewing"

...or I wonder if that belongs in a different topic, because that's not really a git thing.

Reply:
Agreed on leveraging CI/CD as much as possible to help with PR reviews. I think that the testing instructions in the pull request should include any _user_ steps needed to test the feature/fix/whatever. e.g. test login credentials are super helpful for authenticated sites with multiple access roles.

Reply:
Added note about leveraging CI/CD to assist with pull request reviews!